OneCommunityGlobal/HighestGoodNetworkApp
# Peterson implement improvements in the my team filter

# Description
This pull request has been opened to improve the filter of the team in the dashboard section and Leaderboard table.

## Related PRS (if any):
None

## Main changes explained:
The Leaderboard component have been modified to improve the filter of the team in the dashboard section and Leaderboard table.

## How to test:
1. check into current branch
2. do npm install, git pull and ... to run this PR locally
3. Clear site data/cache
4. Log in as an administrator or owner.
5. go to dashboard
6. Check if the tables 'Leaderboard' and 'Team Member Tasks' are filtered simultaneously; the filter features should be easy to use. Check if the member numbers are displayed correctly in the header of the 'Leaderboard' table after the table is filtered.

## Screenshots or videos of changes:
![Peterson_implement_improvements_in_the_My_Team_filter](https://github.com/user-attachments/assets/ac4f947c-b666-468b-a1f0-8dcb46b9b67e)


## Note:
None
